Makefile.am

```diff
@@ -38,7 +38,7 @@ dist_doc_DATA = \
 	COPYRIGHT.GPL \
 	COPYING
 
-AM_CPPFLAGS = $(OPENSSL_CRYPTO_CFLAGS) -D_UNICODE
+AM_CPPFLAGS = $(CRYPTO_CFLAGS) -D_UNICODE
```

Comment:
I reverted that part. Do not pay attention to that.

I realised that it will break openvpn/openvpn-build